extensions/docs/msal-node-extensions.md

```diff
@@ -11,17 +11,64 @@ Supported platforms are Windows, Mac and Linux:
 - Linux - LibSecret is used for storing to "Secret Service" through npm keytar.
 
 ## Code
+
 ### Creating the persistence layer
-Creating the persistence will differ based on what platform you are targeting.
 
+API for creating the persistence layer will differ based on what platform you are targeting.
+
+Alternatively, you can use the `createPersistence` API from [PersistenceCreator](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/extensions/msal-node-extensions/src/persistence/PersistenceCreator.ts) as it's a generic wrapper and selects the appropriate persistence method based on the platform/OS. 
+
+```js
+const { PublicClientApplication } = require("@azure/msal-node");
+const {
+  DataProtectionScope,
+  PersistenceCreator,
+  PersistenceCachePlugin,
+} = require("@azure/msal-node-extensions");
+
+const persistence = await PersistenceCreator.createPersistence({
+                cachePath: "path/to/cache/file.json",
+                dataProtectionScope: DataProtectionScope.CurrentUser,
+                serviceName: "test-msal-electron-service",
+                accountName: "test-msal-electron-account",
+                usePlaintextFileOnLinux: false,
+          });
+// Pass the persistence to msal config's cachePlugin
+msalConfig.cache.cachePlugin = new PersistenceCachePlugin(persistence);
+// Use the persistence object to initialize an MSAL PublicClientApplication with cachePlugin
+const pca = new PublicClientApplication({
+                auth: {
+                        clientId: "CLIENT_ID_HERE",
+                    },
+                cache: {
+                        cachePlugin: new PersistenceCachePlugin(persistence);
+                    },
+                });
+```
+
+Alternatively, you can use below platform-specific options:
 #### Windows:
 ```js
-import { FilePersistenceWithDataProtection, DataProtectionScope } from "@azure/msal-node-extensions";
+
+const { FilePersistenceWithDataProtection, DataProtectionScope } = require("@azure/msal-node-extensions");
```

Comment:
Just curios: why did you change from import to require?

Reply:
to make it consistent with js electron sample